### PR TITLE
Sample code to handle notifications message

### PR DESCRIPTION
**Pull request by olegz-circle**

### CODE REVIEW

File: devtools/node/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "devtools",
+  "version": "1.0.0",
+  "description": "#### HTTP server for local debug",
+  "main": "httpserver.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (mboorstin-circle) [inline devtools/node/package.json:10]: Is this the same license as the license in the main sample-app package.json?

File: devtools/node/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
+    }
+}
```
Comment (mboorstin-circle) [inline devtools/node/.eslintrc.json:19]: Missing newline

File: devtools/node/package.json (hunk 1 above)
Comment (olegz-circle) [reply devtools/node/package.json:10]: There is no license in main sample-app

File: devtools/node/.eslintrc.json (hunk 2 above)
Comment (olegz-circle) [reply devtools/node/.eslintrc.json]: Fixed.

File: devtools/node/package.json (hunk 1 above)
Comment (olegz-circle) [reply devtools/node/package.json]: Added MIT to both

**Pull request merged**